apps/webstore/src/app/pages/auth/signup/signup.component.html

```diff
@@ -56,8 +56,11 @@ <h1 class="page-title" i18n="@@signup">Sign up</h1>
             </button>
         </div>
         <div class="info-message" i18n="@@info-password">
```

Comment:
FYI: This part needs to be translated properly. So this issue cannot be approved before i do translations.

Reply:
I'm also doing the reset password page, so it's not quite complete yet.

Reply:
I've done the reset password page now and requested a re-review. 
[https://gyazo.com/7c53d67d660e5a17a5e756deb3721ed6](url)
[https://gyazo.com/6d03515ce536f7955233b1da9980a248](url)

Do you want the styling changed from how this looks or are you happy to go ahead with translations?

Reply:
I cant see the screenshots. Just add them in the description ;)

Reply:
![image](https://user-images.githubusercontent.com/75076281/188862920-2b387b7d-e6c1-423f-8771-1a5ca3d498e6.png)
![image](https://user-images.githubusercontent.com/75076281/188862937-e9cf848e-e2f9-440e-86ba-3579db91a572.png)

Reply:
Resolved : [e3c6d7d](https://github.com/treecreate/webstore/pull/377/commits/e3c6d7d1747d804aa52db8d9ba6b1e69a4c0fdc3)